components-core/src/main/java/com/adyen/checkout/components/core/PaymentMethodTypes.kt

```diff
@@ -94,6 +95,13 @@ object PaymentMethodTypes {
         listOf(
             ACH,
             BCMC,
+            BOLETOBANCARIO,
```

Comment:
With this PR Boleto is not fully supported yet, would be better to leave it in the unsupported list and then add them here with the voucher PR